src/backend/src/modules/configuration/configuration.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common';
+import * as fs from 'fs';
+@Injectable()
+export class ConfigurationService {
+  private filePath: string;
+
+  private configContents: string;
+
+  private configuration: JSON;
```

Comment:
Is JSON really a type? 🤔 If I were you, I would do something like this. 

```suggestion
  private configuration: Record<string, any>;
```

Reply:
Return type of parse is `any` but I think upcasting it to`Record<string, any>` is fair